docs/markdown-converter.md

```
@@ -0,0 +1,428 @@
+
+**入力:**
+```markdown
+# Diagram
+
+```mermaid
+flowchart TD
+    A --> B
+```
+
+This is a diagram.
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)